python/revng/daemon/schema.graphql

```diff
@@ -53,6 +53,11 @@ type DocumentErrorReason {
 }
 
 type Subscription {
+    invalidations: Invalidation!
+}
+
+type Invalidation {
+    index: BigInt!
```

Comment:
Index is to vague. We need a name that makes what it is obvious.

Reply:
IMHO in the context of an invalidation it's obvious what the index is about, in the same way as the `index` parameter of `runAnalysis`

Reply:
I'm not sure I agree, it might be the index of the invalidation.

In `runAnalysis` it's the index we *expect* to be in remote, not just any index.

Reply:
> I'm not sure I agree, it might be the index of the invalidation.

That's the thing, invalidations are (currently) tied to the index, so `1 index bump -> 1 invalidation`. There isn't a separate ordering.

Reply:
I see, but the "index of invalidation events" is a confusing concept. I'd expect the "index of an event" to be zero at the first event, but this is not the case (unless you just opened the project).

I don't see much benefit in having two similar concept with the same name with slightly different semantics.
Let's stick to one concept only: the index of the commit on globals, so we're always sure what we're talking about.

Reply:
I think there might be a misunderstanding: (in my understanding of the mechanism) there was never, and there will never be a separate index for invalidations. Invalidations happen because the context index has increased, period. The payload of an invalidation basically says to (e.g. `revng-ui`): "We're at global index $index, if you were at $index-1 here's the $invalidation list of target you need to invalidate to stay in sync"

Reply:
I'm just saying, let's never talk about "invalidation event index", let's just talk about "globals commit index".

I understand they would go in parallel, but what's would be weird to me is that on the client the "invalidation index" on the third run of `revng-daemon` will not be `0` (which would make sense being the first "invalidation event" I get) but some other number.

Reply:
Ok, I've renamed the `index` field of the invalidation to `commitIndex` just to be extra-clear